aim/sdk/adapters/pytorch_lightning.py

```diff
@@ -99,7 +99,8 @@ def close(self) -> None:
 
     @property
     def save_dir(self) -> str:
-        return self.experiment.repo.path
+        from aim.sdk.repo import Repo
+        return self._repo_path or Repo.default_repo_path()
```

Comment:
Aim will create a sub-directory (`.aim` by default) and use it as a path to store all tracked data. Hence the `save_dir` should refer to that directory. Here is a code snippet fixing that:
```
import os

from aim.sdk.repo import Repo
from aim.sdk.utils import clean_repo_path, get_aim_repo_name

repo_path = clean_repo_path(self._repo_path) or Repo.default_repo_path()
return os.path.join(repo_path, get_aim_repo_name())
```

Reply:
I test the code above with no problem and use them for pull requests.